gcloud-java-storage/src/main/java/com/google/gcloud/storage/Storage.java

```diff
@@ -69,6 +72,104 @@ String entry() {
     }
   }
 
+  public static abstract class EntityField {
+
+    private final String selector;
+
+    public EntityField(String selector) {
+      this.selector = selector;
+    }
+
+    public String selector() {
+      return selector;
+    }
+  }
+
+  enum BucketField {
+    ID("id"),
```

Comment:
It would be nice to add to IT test a get with explicitly adding all fields (even if we don't check the response), as a typo will be detected there and thrown as an error.

Same for BlobField.